app/src/test/java/org/jellyfin/androidtv/preference/PreferenceStoreTest.kt

```diff
@@ -0,0 +1,96 @@
+package org.jellyfin.androidtv.preference
+
+import io.kotest.core.spec.style.FunSpec
+import io.kotest.core.test.TestCase
+import io.kotest.matchers.shouldBe
+
+/**
+ * Tests that the generic set/get methods dispatch to the correct
+ * internal getT/setT methods
+ */
+class PreferenceStoreTest : FunSpec() {
+	class TestStub : PreferenceStore() {
```

Comment:
Move out of PreferenceStoreTest (but in the same file) and mark private.

Reply:
Huh didn't know you could mark a class private in Kotlin. That's good to know :+1: